cmd/podman/pod_create.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var podCreateDescription = "Creates a new empty pod. The pod ID is then" +
+	" printed to stdout. You can then start it at any time with the" +
+	" podman pod start <pod_id> command. The pod will be created with the" +
+	" initial state 'created'."
+
+var podCreateFlags = []cli.Flag{
+	cli.BoolTFlag{
+		Name:  "cgroup-to-ctr",
+		Usage: "Tells containers in this pod to use the cgroup created for the pod",
+	},
+	cli.StringFlag{
+		Name:  "cgroup-parent",
+		Usage: "Optional parent cgroup for the pod",
+	},
+	cli.StringSliceFlag{
+		Name:  "label-file",
+		Usage: "Read in a line delimited file of labels (default [])",
+	},
+	cli.StringSliceFlag{
+		Name:  "label, l",
```

Comment:
Kube calls them annotations not labels, but it's still just a map[string]string. I think it's probably fine to have labels and annotations be the same thing, since labels matches the container API.

Reply:
could do an alias?

Reply:
@mheon @rhatdan turns out crictl has both. Per api docs:
    // Labels are key-value pairs that may be used to scope and select individual resources.
    map<string, string> labels = 7;
    // Unstructured key-value map holding arbitrary metadata.
    // Annotations MUST NOT be altered by the runtime; the value of this field
    // MUST be identical to that of the corresponding PodSandboxConfig used to
    // instantiate the pod sandbox this status represents.
    map<string, string> annotations = 8;

Reply:
Huh. Alright.

We can add annotations easily enough... I don't know if we want to leave that in CRI-O and not here though.

I vote to leave the labels stuff as is.

Reply:
I opted to leave labels in, and think we should look into adding annotations as k8s does later